apps/src/lib/levelbuilder/CourseOfferingEditor.jsx

```diff
@@ -534,6 +555,10 @@ const styles = {
     paddingRight: '10px',
     cursor: 'pointer',
   },
+  flexContainer: {
+    display: 'flex',
+    alignItems: 'center',
+  },
```

Comment:
Nit: not sure if this is just on my end, but it looks like the little calendar icon is tucked underneath the date picker. I was curious if you knew why this was or if there's a way to easily fix that? If there's no easy fix, it's not a big deal
![Capture](https://github.com/code-dot-org/code-dot-org/assets/56283563/491a9f47-18b7-4c12-938f-b65265663112)